### DISCUSSION TITLE
ICON_SERVICE does not take effect

### DISCUSSION THREAD

**Discussion by DukeFox** [dani-garcia/vaultwarden#2964] [Q&A]
sorry for my bad english
I am using the latest image
“ICON_SERVICE” is set to “bitwarden”
![](https://pic3.58cdn.com.cn/nowater/webim/big/n_v2a3bfc1c0bc3d438ea2d430e8fabbf345.png)
But it didn't work, and the page still use internal to get icon。

**Comment by BlackDex** [comment]
Works fine for me.
What is the reason you think it is not working?
Because calls in all cases go to your own instance first, and then perform a redirect to the selected Icon Service.
